tools/ci_build/build.py

```diff
@@ -325,6 +326,7 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
                  "-Donnxruntime_CUDNN_HOME=" + (cudnn_home if args.use_cuda else ""),
                  "-Donnxruntime_CUDA_HOME=" + (cuda_home if args.use_cuda else ""),
                  "-Donnxruntime_USE_JEMALLOC=" + ("ON" if args.use_jemalloc else "OFF"),
+                 "-Donnxruntime_USE_MIMALLOC=" + ("ON" if args.use_mimalloc else "OFF"), # TODO: error if try to use both jemalloc & mimalloc
```

Comment:
Let's temporarily make this option always ON, so that we can let the CI build system test if your new code works. 
When all the tests all passed, we can revert it back.